api/users/tests/test_models.py

```diff
@@ -173,47 +166,6 @@ def test_has_organisation_permission_is_false_when_user_does_not_have_permission
         )
 
 
-@pytest.mark.django_db
-@pytest.mark.parametrize(
-    "user_permission_keys, group_permission_keys, expected_keys",
-    (
-        ([], [], set()),
-        ([CREATE_PROJECT], [], {CREATE_PROJECT}),
-        ([], [CREATE_PROJECT], {CREATE_PROJECT}),
-        ([CREATE_PROJECT], [CREATE_PROJECT], {CREATE_PROJECT}),
-    ),
-)
-def test_get_permission_keys_for_organisation(
```

Comment:
I'm not sure why this test was removed?

Reply:
Because the method `get_permission_keys_for_organisation` was removed in favour of the permission calculator module